staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -130,14 +141,24 @@ func NewController(netIfc kosclientv1a1.NetworkV1alpha1Interface,
 	workers int,
 	hostname string) *IPAMController {
 
-	attachmentCreateToLockHistogram := prometheus.NewHistogram(
+	addressContentionHistogram := prometheus.NewHistogram(
```

Comment:
Since `attachmentCreateToAddressHistograms` now has a label on whether there was address contention the associated `_count` metric captures the exact same information as this histogram, which can be removed.

Reply:
Or not, `addressContentionHistogram` tracks contentions in real-time, while `attachmentCreateToAddressHistograms` only after the previously-contending NA receives an address.

Reply:
There is another difference.  The addressContentionHistogram gets an observation after every _attempt_ to allocate an address, and there can be multiple attempts (even multiple failed attempts) for one NetworkAttachment.